.circleci/config.yml

```diff
@@ -200,6 +225,9 @@ jobs:
       - checkout
       - *dependencies
       - run: mkdir -p /tmp/logs
+      - restore_cache:
```

Comment:
It's a bit too much of restore_cache, should happen only in the required job, in this case setup_dependencies

Reply:
and when installing/downloading occurs. 
I'll double check now to be sure which of those should keep it

Reply:
`setup_dependencies` is a job that runs only once. Other jobs need to restore the cache indipendently.

Reply:
but are all of them using it?

Reply:
Yes